test/Microsoft.ML.PerformanceTests/Harness/Configs.cs

```diff
@@ -60,9 +62,7 @@ private IToolchain CreateToolchain()
 
         protected static string GetBuildConfigurationName()
         {
-#if NETCOREAPP3_1
-            return "Release-netcoreapp3_1";
-#elif NET461
+#if NETFRAMEWORK
             return "Release-netfx";
 #else
             return "Release";
```

Comment:
>Release [](http://example.com/codeflow?start=20&length=7)

You don't think we need to append this with some specific version? I know this work fine but if we'll have more targets in the future can cause issues?

Reply:
If we need to change in the future we can.   We should try to remove the custom harness entirely.